xocto/storage/storage.py

```diff
@@ -1179,6 +1188,14 @@ def get_size_in_bytes(self, *, s3_object: S3Object) -> int:
         file_stats = os.stat(filepath)
         return file_stats.st_size
 
+    def get_last_modified(self, key_path: str) -> datetime.datetime:
+        file_path = os.path.join(self.storage_root, self.bucket_name, key_path)
+        if not os.path.exists(file_path):
+            raise KeyDoesNotExist(f"Key {key_path} was not found at {file_path}")
+
+        file_stats = os.stat(file_path)
+        return datetime.datetime.fromtimestamp(file_stats.st_mtime)
```

Comment:
ah now I get it this is in `LocalFileStore` so this works, sorry missed that before.

Reply:
Yup, I had to go digging too, hence my comment.